sdk/web-pubsub/test-resources.json

```diff
@@ -95,18 +95,6 @@
     }
   ],
   "outputs": {
-    "AZURE_TENANT_ID": {
-      "type": "string",
-      "value": "[parameters('tenantId')]"
-    },
-    "AZURE_CLIENT_ID": {
-      "type": "string",
-      "value": "[parameters('testApplicationId')]"
-    },
-    "AZURE_CLIENT_SECRET": {
-      "type": "string",
-      "value": "[parameters('testApplicationSecret')]"
```

Comment:
I ran the [live pipeline](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3977609&view=results) against this PR and it is failing at the deploy test resource step. Please remove the `testApplicationSecret` in the `parameters` section at the beginning of the file too. Thank you!

Reply:
Thanks so much @minhanh-phan. I've already removed the testApplicationSecret and the pipeline passes now. That would be enough right?

Reply:
Thank you for the change. I have approved the PR!